UKHomeOffice/cerberus-service
# COP-10599: Add tests to verify different selectors on 2 different versions

## Description
Add tests to verify different selectors on 2 different versions 

## To Test
npm run cypress:runner

run tests from spec `airpax-task-details.spec.js` 
`Should verify the selector matches with same & different group reference on 2 different version of a task`

## Developer Checklist

\* Required
